packages/jest-message-util/src/index.ts

```diff
@@ -38,6 +38,7 @@ export type StackTraceConfig = Pick<
 
 export type StackTraceOptions = {
   noStackTrace: boolean;
+  noCodeFrame?: boolean;
```

Comment:
Would be nice to extend `messages.test.ts` by a small test to cover the new option so that we don't only notice it in seemingly random e2e tests should it break :upside_down_face:

Reply:
pushed 🙂